src/client/pages/Offer/Checkout/inputFields.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { FormEvent } from 'react'
+import { FormikProps, getIn } from 'formik'
+import { CoreInputFieldProps, RawInputField } from 'components/inputs'
+import { useTextKeys } from 'utils/textKeys'
+import { useCurrentLocale } from 'l10n/useCurrentLocale'
+import { QuoteInput } from '../Introduction/DetailsModal/types'
+
+interface CheckoutInputProps {
+  field?: CoreInputFieldProps
+  formikProps: FormikProps<QuoteInput>
+  name: string
+  errors?: string
+  showErrorIcon?: boolean
+}
+
+export const CheckoutInput: React.FC<CheckoutInputProps &
```

Comment:
[Suggestion]
As we're accepting intrinsic input props, spread instead of list them would enable pass down any intrinsic props without having to change the code. E.g: if we want to provide `disabled` to this field.

```suggestion
export const CheckoutInput: React.FC<CheckoutInputProps & React.InputHTMLAttributes<HTMLInputElement>> =  ({
    field,
    formikProps,
    ...props
}) => {
    const textKeys = useTextKeys()
    
    return field ? (
        <RawInputField
           {...props}
            label={textKeys[field.label]()}
            placeholder={textKeys[field.placeholder ? field.placeholder : '']()}
            type={field.type}
            value={getIn(formikProps.values, props.name)}
            onChange={onChange || formikProps.handleChange}
            showErrorIcon={getIn(formikProps.errors, props.name)}
        />
    ) : null
}
```

Reply:
We agree :)